src/index.ts

```diff
@@ -2,16 +2,18 @@
  * These functions must run in the browser, do not use Node APIs.
  */
 
-import slugify from "./slugify";
+import { slugify } from "./slugify/index";
+import { removeSpecialCharacters } from "./handleSpecialCharacters/index";
 
-export { slugify };
+export { slugify, removeSpecialCharacters };
 
 // When bundled for the browser, this object
 // will be exposed on the Window object as
 // `OakRetoolHelpers`.
 const oakRetoolHelpers = {
   slugify,
```

Comment:
Does `removeSpecialCharacters`  need to be added to this to be able to use it in Retool?

Reply:
Yes! thanks Dorothy